Sample/UIKit/CustomEventViewController.swift

```diff
@@ -157,3 +149,25 @@ extension CustomEventViewController: UITextFieldDelegate {
         return false
     }
 }
+
+enum AttributeTypeKeys: String, CaseIterable, Identifiable {
+    case string = "String"
+    case boolean = "Boolean"
+    case integer = "Integer"
+    case double = "Double"
+    case date = "Date (ms)"
+    case none = ""
+
+    var id: String { self.rawValue }
+}
+
+extension AttributeTypeKeys {
+    var text: String {
+        get {
+            return self.rawValue
+        }
+        set {
+            self = AttributeTypeKeys(rawValue: newValue)!
```

Comment:
Is this needed because of Identifiable? Is there any way to avoid this?
Like encapsulate this inside `EventAttribute` struct you created?

Reply:
Yes, it is indeed for Identifiable.
I move this code to EventAttribute [cb224f7](https://github.com/rakutentech/ios-inappmessaging/pull/278/commits/cb224f744aa1783b19604162a63939b77d27fc41).